src/components/AuthLogin/UabcHeaderText.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const UabcHeaderText = () => {
+  return (
+    <div className="text-center">
+      <p className="font-proxima text-[4rem] font-bold leading-[2.5rem] text-primary">
+        UABC
+      </p>
```

Comment:
```suggestion
      <span className="font-proxima text-[4rem] font-bold leading-[5rem] text-primary tracking-tight">
        UABC
      </span>
```
changed style and line height to match figma